server/proxy/enable.go

```diff
@@ -23,15 +22,9 @@ func (p *Proxy) EnableApp(sessionID, actingUserID string, cc *apps.Context, appI
 		return md.MD(fmt.Sprintf("%s is already enabled", app.DisplayName)), nil
 	}
 
-	conf := p.conf.GetConfig()
-	asAdmin, err := utils.ClientFromSession(p.mm, conf.MattermostSiteURL, sessionID, actingUserID)
+	_, err = client.EnableBot(app.BotUserID)
```

Comment:
Are we creating a new bot for plugin apps? I guess we shouldn't, but not 100% sure.

Reply:
Well, the bot name is based on the `AppID`. If the plugin wants to use the same bot account, it needs to use an appropriate ID.

Reply:
Does this mean a plugin's app id must be equal to its existing bot's username?

Reply:
> Does this mean a plugin's app id must be equal to its existing bot's username

If it wants to use the bot, yes. 

I'm hesitant to introduce more complexity by adding bonus features just for apps run by plugins.